Remotion/Web/Test.Shared/ExecutionEngine/ExceptionHandling/MissingUserControlFunction.cs

```diff
@@ -28,6 +28,6 @@ public MissingUserControlFunction ()
     {
     }
 
-    private WxePageStep Step1 = new WxePageStep ("~/ExecutionEngine/ExceptionHandling/MissingUserControlForm.aspx");
+    private WxeResourcePageStep Step1 = new WxeResourcePageStep (typeof (MissingUserControlFunction), "ExecutionEngine/ExceptionHandling/MissingUserControlForm.aspx");
```

Comment:
```suggestion
    private WxeResourcePageStep Step1 = new WxeResourcePageStep (typeof (MissingUserControlForm), "ExecutionEngine/ExceptionHandling/MissingUserControlForm.aspx");
```

please recheck the others, too, there's several Function instead of Form errors.